### DISCUSSION TITLE
Is UnsteadyRANS simulation possible?

### DISCUSSION THREAD

**Discussion by AOE-khkhan** [mdolab/dafoam#319] [General]
I am trying to run Unsteady RANS analysis for DPW4 aircraft. https://dafoam.github.io/mydoc_tutorials_aero_dpw4.html
But not understanding what options need to be set up or changed? Can you give any script for the setup? 
Thanks

**Comment by friedenhe** [comment]
No, we don't support unsteady RANS for transonic flow in DAFoam, but you can do that with OpenFOAM.
